Dockerfile.builder

```diff
@@ -20,7 +20,7 @@ RUN yum --enablerepo=centosplus install -y \
 
 # Get custom go v
 RUN cd /tmp \
-    && wget https://storage.googleapis.com/golang/go1.9.2.linux-amd64.tar.gz  \
+    && wget https://storage.googleapis.com/golang/go1.10.4.linux-amd64.tar.gz  \
```

Comment:
For security reasons I suggest to do a checksum check as we do in fabric8-wit:

```Dockerfile
RUN test -n $USE_GO_VERSION_FROM_WEBSITE \
    && cd /tmp \
    && wget --no-verbose https://dl.google.com/go/go1.10.linux-amd64.tar.gz \
    && echo "b5a64335f1490277b585832d1f6c7f8c6c11206cba5cd3f771dcb87b98ad1a33  go1.10.linux-amd64.tar.gz" > checksum \
    && sha256sum -c checksum \
    && tar -C /usr/local -xzf go1.10.linux-amd64.tar.gz \
    && rm -f go1.10.linux-amd64.tar.gz
ENV PATH=$PATH:/usr/local/go/bin
```

Here we also don't set any `GOROOT` btw.

Reply:
Good point. We should check the checsum.